adala/utils/gsheet.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import gspread
+from google.auth import default
+
+
+def get_client():
```

Comment:
If my assumption that this would require a public sheet is correct I think we may want to add auth creds soon.

Reply:
Every gsheet access will require auth - this one uses default credentials (i.e. accessing from GCP, colab or locally with creds stored in environmental variables)